Documentation/ApiOverview/Rte/InTheFrontend/Introduction.rst

```diff
@@ -0,0 +1,116 @@
+.. include:: /Includes.rst.txt
+.. index::
+   pair: Rich text editor; Frontend
+.. _rte-frontend-introduction:
+
+==================================================
+Including a Rich Text Editor (RTE) in the frontend
+==================================================
+
+When you add forms to a website you might want to offer
+formatting options like bold, italic etc.. Rich Text Editors offer
+extensive options that are configurable for individual needs.
+
+This chapter outlines conceptual and technical information
+about adding an RTE on frontend pages.
+
+.. figure:: ../Images/RteBackend.png
+   :alt: A RTE in the TYPO3 BE
+
+   The rtehtmlarea RTE activated in the TYPO3 backend
```

Comment:
rtehtmlarea  is an old, outdated extension (https://extensions.typo3.org/extension/rtehtmlarea) let's use the rte_ckeditor here.

Reply:
That is the (wrong) caption of the image.  
I copied it from the backend chapter and it has to be replaced all together.  
So you're right but it's on the TODO-list anyway ;-)